### PR TITLE
Alembic Scripts for Address Migration #355

### PR DESCRIPTION
**Pull request by gh2os**
Alembic script that creates the address, provincial state type, and country type tables.

### CODE REVIEW

File: ppr-api/migrations/versions/6411b673d98b_create_address_tables.py [hunk 1]
```diff
@@ -0,0 +1,389 @@
...
+        {'country_type_cd': 'YU', 'description': 'YUGOSLAVIA'},
+        {'country_type_cd': 'ZA', 'description': 'SOUTH AFRICA'},
+        {'country_type_cd': 'ZM', 'description': 'ZAMBIA'},
+        {'country_type_cd': 'ZW', 'description': 'ZIMBABWE'}
+    ])
+
+    prov_state_type_table = op.create_table(
+        'prov_state_type',
+        sa.Column('province', sa.CHAR(length=2), primary_key=True),
+        sa.Column('country_type_cd', sa.CHAR(length=2), sa.ForeignKey('country_type.country_type_cd')),
```
Comment (jguertin) [inline ppr-api/migrations/versions/6411b673d98b_create_address_tables.py:287]: province code is unique within CA and US, just wondering if country_type_cd should be included as a composite PK?  Theoretically yes, but maybe cross that bridge when we come to it?

File: ppr-api/migrations/versions/6411b673d98b_create_address_tables.py [hunk 2]
```diff
@@ -0,0 +1,389 @@
...
+def upgrade():
+
+    country_type_table = op.create_table(
+        'country_type',
+        sa.Column('country_type_cd', sa.CHAR(length=2), primary_key=True),
+        sa.Column('description', sa.String(length=75))
+    )
+
+    op.bulk_insert(country_type_table, [
+        {'country_type_cd': 'AD', 'description': 'ANDORRA'},
```
Comment (jguertin) [inline ppr-api/migrations/versions/6411b673d98b_create_address_tables.py]: Just wondering about case for description.  All upper makes sense in the IMS system, wondering if it's appropriate going forward? It's a lot of effort to change, so maybe leave as is for now?

**Review by jguertin — APPROVED** [review]
Looks good, a couple of questions for consideration, but not blocking.

File: ppr-api/migrations/versions/6411b673d98b_create_address_tables.py (hunk 1 above)
Comment (gh2os) [reply ppr-api/migrations/versions/6411b673d98b_create_address_tables.py:287]: Ah, I see what you mean. Happy to cross the bridge when we get to it, or we can look into addressing that now. I am indifferent.

**Review by WalterMoar — APPROVED** [review]

**Review by jguertin — APPROVED** [review]

**Pull request merged**